scripts/mine-block.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```

Comment:
I think this can be accomplished using `cast rpc anvil_mine ${n}`
https://book.getfoundry.sh/reference/anvil/?highlight=anvil_mine#description